### ISSUE TITLE
Verify code for Slot filling using RNN

### ISSUE THREAD

**Issue by Jisoo-Min** [jiisoo/NLP-FindLoop#4]
Link : https://github.com/chsasank/ATIS.keras

We have to check that it's the same code we intended. (Slot filling)

**Comment by Jisoo-Min** [comment]
When this code runs, it shows progress bar while training.
![image](https://user-images.githubusercontent.com/24731075/61833118-330e2280-ae28-11e9-8a60-64f40fc9556e.png)


While training, it show precision, recall, and f1-score.
each of them gets high score.
![image](https://user-images.githubusercontent.com/24731075/61833033-f2160e00-ae27-11e9-9dbf-09bfa95d4489.png)

This code records the maximum value of the f1-score. 
<img src="https://user-images.githubusercontent.com/24731075/61833218-710b4680-ae28-11e9-9fc3-89a836c6eb0f.png" width="50%">




The maximum f1-score in the paper "Using Recurrent Neural Networks for Slot Filling in Spoken Language Understanding" is 95.04, and it is from Elman-RNN with stochastic(STO)

The maximum f1-score I got is 94.35. It seems to be reasonable :) 
<img src="https://user-images.githubusercontent.com/24731075/61833427-20e0b400-ae29-11e9-9563-4b07b09dd8d6.png"  width="50%">


Now, I have to analyze and understand the code. 
To make it the same as we intended, we need to find out what parameters can be adjusted.

**Comment by Jisoo-Min** [comment]
# About Input

**Format of Train-set and Test-set** 
![image](https://user-images.githubusercontent.com/24731075/61851352-8fdefc80-ae6b-11e9-9b77-3fce0de2f079.png)

###  Additional data to use: Labels to index 

1. 
<img src="https://user-images.githubusercontent.com/24731075/61851418-c6b51280-ae6b-11e9-83a8-d8228f1996ee.png" width="60%">

2. 
<img src="https://user-images.githubusercontent.com/24731075/61851428-d0d71100-ae6b-11e9-816d-6b6eb8e7b3a7.png" width="50%">

3. word embeddings
<img src="https://user-images.githubusercontent.com/24731075/61851434-d7658880-ae6b-11e9-90aa-b9b2c9f16376.png" width="60%">

**Comment by jiisoo** [comment]
It seems not to use RNN 
the paper shows RNN with Jordan and Elman, but this code do not use it

**Comment by jiisoo** [comment]
dd
